ngraph_bridge/ngraph_builder.cc

```diff
@@ -159,7 +159,7 @@ static Status GetInputNode(const Builder::OpMap& ng_op_map, const Node* op,
 
   Node* tf_input;
   TF_RETURN_IF_ERROR(op->input_node(input_idx, &tf_input));
-  const std::vector<shared_ptr<ng::Node>>* ng_op = nullptr;
+  const std::vector<ng::Output<ng::Node>>* ng_op = nullptr;
   try {
     ng_op = &ng_op_map.at(tf_input->name());
```

Comment:
```suggestion
  std::vector<ng::Output<ng::Node>> ng_op;
  try {
    ng_op = ng_op_map.at(tf_input->name());
```
This will not result in a copy of the vector due to return value optimization